tests/Unit/PointwiseFunctions/AnalyticSolutions/GrMhd/Test_KomissarovShock.cpp

```diff
@@ -183,6 +192,23 @@ void test_variables(const DataType& used_for_size) noexcept {
       {{{-1., 1.}}}, member_variables, used_for_size);
 }
 
+void test_solution() noexcept {
+  grmhd::Solutions::KomissarovShock solution(
+      1.33, 1., 3.323, 10., 55.36, std::array<double, 3>{{0.83, 0., 0.}},
+      std::array<double, 3>{{0.62, -0.44, 0.}},
+      std::array<double, 3>{{10., 18.28, 0.}},
+      std::array<double, 3>{{10., 14.49, 0.}}, 0.5);
+  const std::array<double, 3> x{{1.0, 2.3, -0.4}};
+  const std::array<double, 3> dx{{1.e-1, 1.e-1, 1.e-1}};
+
+  domain::creators::Brick<Frame::Inertial> brick(
```

Comment:
Is the domain particularly dependent on the solution being tested? If not I would factor it into the verify_grmhd_solution code.

Reply:
yes, you need to choose a domain that is appropriate for the solution.  After this PR is merged, the authors of the individual solutions should update the tests to verify the solution at both random and appropriate interesting points (e.g. on coordinate axes, the origin, etc) where the solution is smooth